bigtop-packages/src/common/ambari/patch6-support-ppc.diff

```diff
@@ -42,6 +42,14 @@ index 0634988657..022037b552 100644
 +          16,
 +          18
 +        ]
++      },
```

Comment:
This patch was for Powerpc.
It seems openEuler does not support ppc.

Reply:
About support for openEuler has moved from patch6-support-ppc.diff to patch14-support-openeuler.diff. @guyuqi 

code commit: https://github.com/apache/bigtop/pull/1147/commits/4c4db4a2d38e333ae3d15fca951136e9d3cdfdad